specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
I'm not entirely clear what it is for either.  It is a term used in the draft C.R.S. which I think means the *uncorrected* tabulation outcome that is used as an input to the RLA algorithm.